README.adoc

```diff
@@ -0,0 +1,64 @@
+[[minishift-addons]]
+= Minishift Add-ons
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[welcome-to-minishift]]
+== Welcome to the Minishift add-ons repository!
+
+This repository contains add-ons contributed by the Minishift community.
+
+We hope that by sharing these add-ons, folks in the community will be able to make the most out of their Minishift environment.
+Community add-ons are tested and supported by the community.
+
+https://ci.centos.org/job/minishift-addons/[image:https://ci.centos.org/buildStatus/icon?job=minishift[Build Status]]
+
+'''''
+
+[[introduction-addons]]
+== Introduction to add-ons
+
+link:https://docs.openshift.org/latest/minishift/using/addons.html[Minishift add-ons] are extensions, or plug-ins, that customize the default OpenShift cluster configuration and behavior.
+For example, you can write an add-on to modify user permissions for the *oc* binary, or to run commands on the Docker daemon inside the Minishift VM.
+
+Minishift includes several link:https://docs.openshift.org/latest/minishift/using/addons.html#default-addons[default add-ons], which are stored in *_$MINISHIFT_HOME/addons_*.
+These add-ons are tested and supported by the Minishift core team, although due to the experimental nature of the add-ons mechanism, backwards-compatibility is not guaranteed.
```

Comment:
"experimental status" of the add-ons mechanism? Add-ons are not experimental. Also I am not sure what you want to imply with that "backwards-compatibility is not guaranteed".